### PR TITLE
Adding amenity=bbq icon

### LINKED ISSUE #2996: Rendering of amenity=bbq
I have done a fair amount of mapping contributions and I am wondering why the amenity=bbq is not being rendered.

### PR DESCRIPTION
**Pull request by james2432**
Fixes #2996 
From https://www.openstreetmap.org/node/4983372050#map=19/45.35162/-75.82804 (node):
image

From: https://www.openstreetmap.org/#map=19/43.31581/-70.92575 (way)
image
~(Should we render ways with a border? or just the little icon?)~

From: https://www.openstreetmap.org/#map=17/45.51983/-75.84958 (node with text in name)
image

### CODE REVIEW

**Comment by kocio-pl** [comment]
> Should we render ways with a border? or just the little icon?

I guess icon is enough. If this is in the recreation zone, it's easy to add such area in parallel.

File: amenity-points.mss [hunk 1]
```diff
@@ -1906,7 +1913,8 @@
     }
   }
 
-  [feature = 'amenity_atm'][zoom >= 17] {
+  [feature = 'amenity_atm'][zoom >= 17],
+  [feature = 'amenity_bbq'][zoom >= 17] {
     text-name: "[operator]";
```
Comment (kocio-pl) [inline amenity-points.mss]: Name tag is used here, operator is not even visible in stats:

https://taginfo.openstreetmap.org/tags/amenity=bbq#combinations

**Comment by james2432** [comment]
Replaced operator use by name, not sure why I was using operator (was thinking maybe bbq's usually are operated by someone vs have a name?)

**Comment by kocio-pl** [comment]
Nice, could you also render a way with a name?

**Comment by kocio-pl** [comment]
After merging similar item (`amenity=shower`) you need to fix the conflicts in `project.mml` to make this PR mergable again.

**Comment by james2432** [comment]
Fixed.

**Comment by kocio-pl** [comment]
It renders OK and I'm gonna merge it, but could you make it 1 commit (instead of 2)?

**Comment by james2432** [comment]
Done.

File: amenity-points.mss (hunk 1 above)
Comment (kocio-pl) [reply amenity-points.mss]: Something went wrong and - just by looking at code - you render operator instead of name again.

**Comment by kocio-pl** [comment]
Unfortunately we're back to the operator label (instead of a name label).

**Comment by james2432** [comment]
Not sure what went wrong, possibly this computer didn't fetch my origin changes. Sorry about that

**Comment by james2432** [comment]
Fixed the conflict issue with advertising

**Comment by kocio-pl** [comment]
Thanks! 

We are getting more outdoor lately (shower, bbq, fire pit, aerialways, shrines, towers...). That's great, OSM data are really diverse.

**Pull request merged**